linode/manager
# M3-5282: Linode Detail - Networking - Improved stats empty state

## Description

- Improves the stats unavailable state displayed at `/linodes/{id}/networking` when you create a new Linode
  -  Users were getting confused by the old error state and thought something was wrong with their Linode following it provisioning
- Updates this component to use **React Query**

![Screen Shot 2022-04-11 at 8 08 06 PM](https://user-images.githubusercontent.com/6440455/162853284-e49ffba5-ba04-4548-957f-2c2aea0b1f01.jpg)

## How to test

- Create a new Linode and navigate to its `Networking` tab immediately
- Watch this tab for **7 minutes** and verify that, stats eventually show up in the graph
  - During the first 7 minute period, Cloud Manager should not even hit the Linode API for stats
  - After 7 minutes, you may still get the "stats not ready" message, this may be because the API is still not ready
  - Continue checking that stats eventually show up (should start working after about 10 minutes)
  - Keep in mind, because there is no polling, to get Cloud to re-check for stats, you must refresh, or refocus the window to get react query to refetch
- Verify that all functionality of the `Network Transfer History` graph still works
